### PR TITLE
Defined netapp_e_syslog storage module

### PR DESCRIPTION
**Pull request by ndswartz**
##### SUMMARY

Module allows syslog server configuration on NetApp E-Series storage arrays.

This patch also add integration and unit tests.

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
netapp_e_syslog

##### ANSIBLE VERSION

```ansible 2.6.0 (syslog_pub 0f0ff3b450) last updated 2018/07/06 09:07:44 (GMT -500)
  config file = None
  configured module search path = [u'/home/swartzn/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/swartzn/ansible/lib/ansible
  executable location = /home/swartzn/ansible/bin/ansible
  python version = 2.7.15rc1 (default, Apr 15 2018, 21:51:34) [GCC 7.3.0]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -251,7 +251,6 @@ lib/ansible/modules/cloud/linode/linode.py E322
 lib/ansible/modules/cloud/linode/linode.py E324
 lib/ansible/modules/cloud/lxc/lxc_container.py E324
 lib/ansible/modules/cloud/lxc/lxc_container.py E326
-lib/ansible/modules/cloud/lxc/lxc_container.py E210
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt]: Most of the changes to this file appear to be unrelated to the PR.

**Review by lmprice — APPROVED** [review]
shipit

**Comment by lmprice** [comment]
@ndswartz Oops, missed the conflicts. You'll need to get those resolved.
